### DISCUSSION TITLE
Tooltip Background Color

### DISCUSSION THREAD

**Discussion by Lindsay-Mathieson** [microsoft/fluentui-blazor#2240] [Q&A] [answered]
I'd like to set the tooltip beak and body background color, but haven't been able to make the css for it work - any tips/suggestions please?

**Comment by vnbaaij** [comment, marked as answer]
Hi,

You need to target element(s) in the shadow DOM to achieve that:
```
fluent-tooltip::part(tooltip), fluent-tooltip::part(tooltip):after {
    background: lightgreen !important;
}
```

**Reply by Lindsay-Mathieson** [reply]
> fluent-tooltip::part(tooltip), fluent-tooltip::part(tooltip):after {
>     background: lightgreen !important;
> }

Thanks, I tried setting the style directly (and via class), but couldn't make it work.

```html
@page "/"

<PageTitle>Home</PageTitle>

<FluentButton Id="bnTest">Test</FluentButton>

<FluentTooltip Anchor="bnTest"
               Style="fluent-tooltip::part(tooltip), fluent-tooltip::part(tooltip):after {background: lightgreen !important;}"
               Visible="true"
               UseTooltipService="true">
    Remark required
</FluentTooltip>

```

**Reply by dvoituron** [reply]
You cannot use `::part` in the style attribute. 

You have to use a CSS file or a `style` tag:

```css
<style>
fluent-tooltip::part(tooltip), fluent-tooltip::part(tooltip):after {
    background: lightgreen !important;
}
</style>
```

**Reply by Lindsay-Mathieson** [reply]
Thanks! That did it for all the tooltips on the page

Is there anyway I can apply it to individual tooltips? via class or somesuch?

**Reply by vnbaaij** [reply]
Yes. Assign a class to the tooltip and use that one in the css style to select the one you want to target

**Reply by Lindsay-Mathieson** [reply]
Thankyou! Getting a better handle on css now
